### ISSUE TITLE
uiautomator2.JsonRpcError: -32001 Jsonrpc error: <java.lang.ClassCastException> data: java.lang.ClassCastException: java.lang.Object cannot be cast to android.view.accessibility.AccessibilityWindowInfo

### ISSUE THREAD

**Issue by HeartIsBeat** [openatx/uiautomator2#140]
```d(resourceId=id).click_exists(timeout=timeout)```
点击一个tab页面还在加载中，然后有个弹框广告，就采用click_exists去点击。会报错

## Versions (版本)
Version: 0.1.3.dev8

- 手机型号 三星note5

## Screenshots （相关截图）
![image](https://user-images.githubusercontent.com/20472976/41895084-a192a70e-7953-11e8-9b45-373c63cd94c8.png)

**Comment by HeartIsBeat** [comment]
![image](https://user-images.githubusercontent.com/20472976/41896621-ac4137ac-7957-11e8-9e0a-243d612bbf4c.png)

**Comment by HeartIsBeat** [comment]
dev5版本时，apk未升级时，click_exists挺好使的
后面就又不好使用了
![image](https://user-images.githubusercontent.com/20472976/41897202-3935aa5c-7959-11e8-94f0-fed88b2fb948.png)

**Comment by codeskyblue** [comment]
执行前加一段代码试试

```
d.jsonrpc.setConfigurator({"waitForIdleTimeout": 1000})
```

**Comment by codeskyblue** [comment]
出错的手机画面方便截图下不

**Comment by HeartIsBeat** [comment]
![image](https://user-images.githubusercontent.com/20472976/41948989-002ed5c8-79f2-11e8-982a-32ae87ee99cc.png)

**Comment by codeskyblue** [comment]
之前正常的版本号有吗？现在的版本。还有你运行哪一行代码出错的

**Comment by HeartIsBeat** [comment]
正常版本的，Version: 0.1.3.dev5，然后atx-agent应该是在0.3.3以下的很旧的版本。

**Comment by HeartIsBeat** [comment]
com.github.uiautomator 版本 1.0.12那边的

**Comment by mingyuan-xia** [comment]
更新一下com.github.uiautomator apk文件，建议重新init

**Comment by codeskyblue** [comment]
@mingyuan-xia apk你测试了吗？我还没发版呢？
